src/services/accounts/AccountsBalanceInfoService.spec.ts

```diff
@@ -4,12 +4,43 @@
 import { AccountInfo, Address, Hash } from '@polkadot/types/interfaces';
 
 import { sanitizeNumbers } from '../../sanitize/sanitizeNumbers';
+import { polkadotMetadata } from '../../test-helpers/metadata/metadata';
+import { polkadotRegistry } from '../../test-helpers/registries';
+import { createApiWithAugmentations } from '../../test-helpers/typeFactory';
 import { blockHash789629, mockApi, testAddress } from '../test-helpers/mock';
 import accountsBalanceInfo789629 from '../test-helpers/responses/accounts/balanceInfo789629.json';
 import { AccountsBalanceInfoService } from './AccountsBalanceInfoService';
 
 const accountsBalanceInfoService = new AccountsBalanceInfoService(mockApi);
 
+/**
+ * Historical Queries MockApi
+ */
+const historicalMockApi = {
+	query: {
+		balances: {
+			freeBalance: () =>
+				Promise.resolve().then(() => {
+					return polkadotRegistry.createType('Balance', 123456789);
+				}),
```

Comment:
Wouldn't that be easier?

```suggestion
			freeBalance: async () => polkadotRegistry.createType('Balance', 123456789),
```

Reply:
Nice catch, yea I definitely agree, will fix it up now.

Reply:
fixed